x-pack/plugins/cases/public/methods/get_create_case_flyout.tsx

```diff
@@ -35,3 +35,9 @@ export const getCreateCaseFlyoutLazy = ({
     </Suspense>
   </CasesProvider>
 );
+
+export const getCreateCaseFlyoutLazyNoProvider = (props: CreateCaseFlyoutProps) => (
```

Comment:
Why the need to have this?

Reply:
Just read your comment here https://github.com/elastic/kibana/pull/125505/files#r806908363